src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/CommunityApiDataService.kt

```diff
@@ -62,9 +62,6 @@ class CommunityApiDataService(private val communityApiClient: CommunityApiClient
 
   fun isFemaleOffender(crn: String): Boolean {
     return getOffenderGender(crn).equals("Female", true)
```

Comment:
I had to look this up. The true is for ignore case, interesting flag. For readability I would prefer 
getOffenderGender(crn).toLowerCase().equals("female")

Might even lowercase it in getOffenderGender (great name) - I can't imagine we will ever need it to be in whatever original mess it is in feMALe